examples/cugraph-pytorch/cugraph-local/rgcn-hetero/model.py

```diff
@@ -0,0 +1,569 @@
+"""RGCN layer implementation"""
+# File same as: https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn-hetero/model.py
+from collections import defaultdict
```

Comment:
No need to review as same as https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn-hetero/model.py